client/components/select-vertical/index.tsx

```diff
@@ -70,13 +71,18 @@ const SelectVertical: React.FC< Props > = ( {
 				suggestions={
 					! isDebouncing
 						? mapManySiteVerticalsResponseToVertical(
-								( searchTerm === '' ? featured : suggestions ) || []
+								( ! hasUserInput ? featured : suggestions ) || []
 						  )
 						: []
 				}
 				isLoading={ isDebouncing || isLoadingDefaultVertical || isLoadingSuggestions }
+				isShowSkipOption={ hasUserInput }
 				isDisableInput={ isLoadingDefaultVertical }
+				onSuggestionsHide={ () => {
+					setHasUserInput( false );
+				} }
 				onInputChange={ ( searchTerm: string ) => {
+					setHasUserInput( searchTerm !== '' );
```

Comment:
hmm could this be changed to also check if a vertical is selected in order to fix the issue I mentioned with partial searched?